aten/src/ATen/native/LinearAlgebra.cpp

```diff
@@ -2517,25 +2525,65 @@ void compute_T18_scale_square(
   scalar_t theta
 ) {
   // Scale
-  const auto s = at::max(
-    at::zeros_like(norm),
-    at::ceil(at::log2(norm / theta))
-  ).unsqueeze(-1).unsqueeze(-1).to(at::kLong);
-  const auto pow2s = at::pow(2, s);
-  const auto a_scaled = a / pow2s;
+  // We eventually need to do the matrix multiplication to calculate the result.
+  // For exampe, if we have `norm` equal to [27, 6, 6, 0.05], we will end up to
```

Comment:
nit. example

The comments have quite a few errata. Mind fixing those?

Reply:
Oh, sorry, but can you be more clarify?